pkgs/development/libraries/xsimd/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib, stdenv, fetchgit, fetchFromGitHub, cmake, gtest }:
+let
+  version = "7.5.0";
+
+  darwin_src = fetchgit {
```

Comment:
Please use fetchFromGitHub which is more performant as it can download tarballs.